practice-two/src/components/common/Popover.jsx

```diff
@@ -9,19 +9,25 @@ import Button from './Button/Button';
  * @param isOpen - Flag to determine if the popover is open.
  * @param onClosePopover - Function to be called when the popover should be closed.
  * @param closeButton - Flag to determine whether to show the close button.
+ * @param closeButtonContent - Text content for the close button.
+ * @param closeButtonStyle - CSS class for styling the close button.
  * @param popoverClassName - Custom class name for the main popover div.
  * @param arrowPopover - Flag to determine whether to show the arrow in the popover.
  * @param isFixed - Flag to determine whether to use fixed positioning for the popover.
+ * @param fixedOverlayClassName - Custom class name for the fixed overlay element.
  * @returns {JSX.Element|null} The Popover component JSX.
  */
 const Popover = ({
   isOpen = false,
   onClosePopover = () => {},
   closeButton = true,
+  closeButtonContent = 'Close',
+  closeButtonStyle = '',
   children,
   popoverClassName,
   arrowPopover = true,
   isFixed = false,
+  fixedOverlayClassName,
```

Comment:
Not a good solution. it's all about className and style, it's should not stuck in only special case

Reply:
Yes, Im change name fixedOVerlayClassName into customClassNames